api/openapi.yaml

```diff
@@ -1626,6 +1628,46 @@ components:
       #  - asset
       #  - scan
 
+    AssetScanBenchmarks:
```

Comment:
maybe AssetScanStats?
not all the info inside will be regarding benchmarking but also input info